designsafe/apps/rapid/views.py

```diff
@@ -96,7 +96,9 @@ def opentopo_data(request):
         logger.debug("Requesting opentopo data")
         response = requests.get(OPEN_TOPO_REQUEST_FOR_WHOLE_WORLD)
         response.raise_for_status()
-        response_data = response.json()
+        # response has some tabs char in the strings values that need to be replaced with string '\t'
+        response_content = response.text.replace('\t', '\\t')
```

Comment:
I saw the email from the OpenTopo team that the escape character issue should be fixed. Can you confirm that?